python/dgl/graph.py

```diff
@@ -2760,22 +2761,16 @@ def incidence_matrix(self, typestr, ctx=F.cpu()):
     def line_graph(self, backtracking=True, shared=False):
         """Return the line graph of this graph.
 
-        Parameters
-        ----------
-        backtracking : bool, optional
-            Whether the returned line graph is backtracking.
+        See :func:`~dgl.transform.line_graph`.
+        """
+        return dgl.line_graph(self, backtracking, shared)
 
-        shared : bool, optional
-            Whether the returned line graph shares representations with `self`.
+    def reverse(self, share_node_attrs=True, share_edge_attrs=True):
```

Comment:
`share_ndata`, `share_edata`, and they default to be false.

Reply:
Done